spec/spec_helper.rb

```diff
@@ -1,5 +1,30 @@
 # frozen_string_literal: true
 
+# Simplecov needs to be loaded before we require `memo_wise` in order to
+# properly track all memo_wise files
+if Gem.loaded_specs.key?("codecov")
+  require "codecov"
+  require "simplecov"
+
+  SimpleCov.start do
+    enable_coverage :branch
+  end
+
+  SimpleCov.formatter = if ENV["CI"] == "true"
+                          SimpleCov::Formatter::Codecov
+                        else
+                          # Writes coverage file into coverage/index.html
+                          # when run outside of CI for local development
+                          SimpleCov::Formatter::HTMLFormatter
+                        end
+
+  # SimpleCov.refuse_coverage_drop is only implemented for line coverage, so for
+  # branch coverage we must use `minimum_coverage`
+  SimpleCov.minimum_coverage branch: 90
```

Comment:
Out of curiosity, why 90? What's our branch coverage at now?

Reply:
It's at ~93. I figured we could work to improve it and then move this up! The 90 was just arbitrarily lower than what we're currently at. What are you thinking here?

Reply:
I'd love to see this at 100, so whatever you think is the best way to get there works for me!

Reply:
Yup let's keep at 90 in this PR, and when it's fixed we'll change it to 100. Added this to Issue #56.